src/library-authoring/LibraryHome.tsx

```diff
@@ -1,55 +1,61 @@
 import React from 'react';
+import { Stack } from '@openedx/paragon';
 import { useIntl } from '@edx/frontend-platform/i18n';
-import {
-  Card, Stack,
-} from '@openedx/paragon';
 
 import { useSearchContext } from '../search-manager';
 import { NoComponents, NoSearchResults } from './EmptyStates';
 import LibraryCollections from './LibraryCollections';
 import { LibraryComponents } from './components';
+import LibrarySection from './components/LibrarySection';
+import LibraryRecentlyModified from './LibraryRecentlyModified';
 import messages from './messages';
 
-const Section = ({ title, children } : { title: string, children: React.ReactNode }) => (
-  <Card>
-    <Card.Header
-      title={title}
-    />
-    <Card.Section>
-      {children}
-    </Card.Section>
-  </Card>
-);
-
 type LibraryHomeProps = {
   libraryId: string,
+  tabList: { home: string, components: string, collections: string },
+  handleTabChange: (key: string) => void,
 };
 
-const LibraryHome = ({ libraryId } : LibraryHomeProps) => {
+const LibraryHome = ({ libraryId, tabList, handleTabChange } : LibraryHomeProps) => {
   const intl = useIntl();
-
   const {
     totalHits: componentCount,
     searchKeywords,
   } = useSearchContext();
 
   const collectionCount = 0;
 
-  if (componentCount === 0) {
-    return searchKeywords === '' ? <NoComponents /> : <NoSearchResults />;
-  }
+  const renderEmptyState = () => {
+    if (componentCount === 0) {
+      return searchKeywords === '' ? <NoComponents /> : <NoSearchResults />;
```

Comment:
I found a small bug here.

![image](https://github.com/user-attachments/assets/a3e1e604-730d-4f7c-afdd-1bd3e7a2a727)

If we use the `Recently Published` sort and don't have any published components we have `componentCount === 0` with `searchKeywords === ''`, rendering the wrong state.

Maybe we should add an `isFiltered` flag inside our search context to check this condition and use it here.

Reply:
Good catch! Maybe we can adjust the message.. will ask Product.

Reply:
Addressed with [94ac0a1](https://github.com/openedx/frontend-app-course-authoring/pull/1147/commits/94ac0a1a3676c38057043bcb43fa16ca9af62277).